teacher.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Teacher < Person
+  attr_reader :specialization
+
+  def initialize(specialization, age, name = 'Unknown')
```

Comment:
- [ ] The constructor parameters of the `Teacher` class should also include all parameters previously in the `Person` class. .e.g. `def initialize(age, specialization, name = "Unknown", parent_permission = true)`. Kindly fix this